ui/src/settings.ts

```diff
@@ -31,6 +33,22 @@ interface Option<T> {
   title: string;
 }
 
+export enum Theme {
+  Dark = "dark",
+  Light = "light",
+  H4x0r = "h4x0r",
+}
```

Comment:
This is already imported from `./proxy/settings.ts`. You can remove it here.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1754/commits/1cfc90ec9a3d1ed94fabbc9431509d5efde31afc

Reply:
thx